README.md

```diff
@@ -10,20 +10,32 @@ Mainnet Beta with the program ID `recr1L3PCGKLbckBqMNcJhuuyU1zgo8nBhfLVsJNwr5`.
 
 ## Building and Testing
 
-1. Install dependencies:
-
-   ```bash
-   pnpm install
-   ```
+### Program
 
 1. **Build the Program:**
 
-   ```bash
-   pnpm programs:build
-   ```
+```bash
+make build-sbf-program
+```
+
+2. **Run Tests:**
+
+```bash
+make test-program
+```
+
+### JS Client
+
+1. **Build Package:**
+
+```bash
+cd clients/js-legacy
+pnpm i
+pnpm build
+```
 
-1. **Run Tests:**
+2. **Run Tests:**
 
-   ```bash
-   pnpm programs:test
-   ```
+```bash
+make test-js-clients-js-legacy
```

Comment:
Can you get this command to work? On my machine, I seem to get a pattern mismatch. We want to invoke the `test-js-%` command, but this invokes the `test-%` command, which is a rust command. When I moved the `test-js-%` command to appear before `test-%` command in the make file, then I was able to get this to work. But I see that `test-js-%` appears after `test-%` in other repositories as well, so maybe it is just my machine?